aiidalab/__main__.py

```diff
@@ -433,28 +433,27 @@ def install(
         ],
         headers=["App", "Version", "Path"],
     )
-    click.echo(f"Would install:\n\n{indent(apps_table, '  ')}\n")
+    click.echo(f"Installation plan:\n\n{indent(apps_table, '  ')}\n")
     if yes or click.confirm("Proceed?", default=True):
         for app, version in install_candidates.values():
-            if version is not None:
-                if dry_run:
-                    click.secho(
-                        f"Would install '{app.name}' version '{version}'.", fg="green"
-                    )
-                else:
-                    try:
-                        app.install(
-                            version=version,
-                            install_dependencies=dependencies == "install",
-                            python_bin=python_bin,
-                        )
-                    except RuntimeError as error:
-                        click.secho(f"Error: {error}", fg="red")
-                        break
```

Comment:
Well, not exactly the same by changing to raise exception if there are more things going after the for loop ;)